index.bs

```diff
@@ -1696,22 +1696,80 @@ Team Representative in an Interest Group</h5>
+	Prior to [=AC Review=],
+	<dfn>charter drafts</dfn> go through a public [=charter refinement=] phase
+	during which the [=charter=] receives [=wide review=]
+	and the [=Chartering Facilitator=] attempts to find [=consensus=] on the charter.
+	The [=Team=] <em class=rfc2119>may</em> send notice to the [=AC=]
+	when starting development of a new charter prior to the [=charter refinement=] phase.
+
+<h3 id="charter-initiation">
+Initiating Charter Refinement</h3>
+
+	Formal [=charter refinement=] (see below) is initiated
+	by the [=Team=] sending a [=charter review notice=]
+	to the [=Advisory Committee=],
+	to the public,
+	and, in the case of rechartering, to the affected [=Group=].
+
+	This <dfn>charter review notice</dfn> must include:
+	* A short summary of the proposal.
+	* The location of the [=charter draft=], which must be public.
+	* How to participate in the discussion of this [=charter draft=] and where to file issues.
+	* The expected duration of the [=charter refinement=] phase,
+		which must not be less than 28 days,
+		and should not be more than 6 months.
+	* Who the [=Chartering Facilitator=] is.
+
+	The [=Team=] is responsible for initiating [=charter refinement=]
+	at its discretion, in consideration of discussions with the community.
+	An [=Advisory Committee representative=] <em class=rfc2119>may</em> formally request
+	that the [=Team=] initiate [=charter refinement=].
+	The Team <em class=rfc2119>may</em> deny such a request
+	if it thinks the proposal is insufficiently mature or does not align with W3C's scope and mission.
+	This rejection is a [=Team Decision=] which can be <a href="#registering-objections">Formally Objected to</a>.
+
+<h3 id="charter-development" oldids="WGCharterDevelopment">
+Charter Refinement</h3>
+
+	During <dfn>charter refinement</dfn>,
+	the W3C community further develops the [=charter draft=]
+	with the goal of achieving [=consensus=] on the proposal.
+	The <dfn>Chartering Facilitator</dfn>
+	acts as [=Chair=] for the [=charter refinement=] process.
+
+	Note: The [=Chartering Facilitator=] is not necessarily (and often is not) the [=Chair=] of the [=chartered group|group=] being chartered.
+
+	During [=charter refinement=]:
+	* All issues filed against the [=charter draft=] must be [=formally addressed=],
+		and their resolutions tracked in a disposition of comments.
+	* [=Wide review=] of the [=charter draft=] is initiated and completed.
+	* Decisions are made as  [=group decisions=] of all individuals participating in this process.
+		(However note that only W3C [=Members=] and the [=Team=] can particpate in a formal [[#Votes|vote]].)
+
+	The [=charter refinement=] phase concludes when there is either:
+	* A [=group decision=] to initiate [=AC Review=] of the [=charter draft=].
+	* A [=chair decision=] to abandon the proposal.
+
+	Any [=Formal Objection=] filed during the  [=charter refinement=] phase--
```

Comment:
thinking some more about that change: an alternative (based on a proposal @ianbjacobs and I had sketched a few months ago) may be to collect unresolved objections during the charter refinement phase and bring that as input to the AC Review; it would be then up to AC Reps to decide whether these objections deserve to be made formal in the approval decision.

This helps with the following aspects:
* it creates a filter to possibly frivolous objections from any random person reaching the formal decision process
* since the participation in the charter refinement process is not clearly anchored in delegation or representation for Members, it allows for that clarity to be brought back through formal AC reviews

Reply:
the above could also be experimented with without change to the Process Document (which entails both that we can start applying it right away, and that we don't have to encode an untested process quite yet)

Reply:
I don't think an unresolved objection by an AC Rep should be brought as input to the AC Review, that just requires the same AC rep to object twice, which is unnecessary paperwork.

But, threading the needle in between, we could:
* adjust: “Decisions are made as  group decisions of all <del>individuals</del><ins>AC Reps</ins> participating in this process.” or “ of all <del>individuals</del><ins>[W3C Member Reps](https://www.w3.org/policies/process/#member-rep)</ins>”
* keep “Any Formal Objection filed during the  charter refinement phase […] is not considered until the close of the AC Review of the charter”
* keep: “All issues filed against the charter draft must be formally addressed, and their resolutions tracked in a disposition of comments.”

This is close to but not identical to what you said. People who are not part of the decision making body could still object but only post facto. If they do formally object, it lands on the Council, but later. If they do not formally object, but do disagree with the group's decision, it gets tracked in the DoC, which the AC Reps see during the AC Review, and they can then take that into account in terms of

Reply:
I tend to agree with Dom's suggestions here: It would be good to avoid FOs during the refinement phase, and also experiment before including it in the Process. My concern overall is that we introduce a two-stage process in which the AC review is seen as having less validity, should FOs occur at AC review, because the group of participants in the refinement phase came to an agreement among themselves.

Reply:
The Process CG has [decided to fold this text as is into the ED](https://www.w3.org/2024/07/24-w3process-minutes.html#r09) for now, for a few reasons:
* Chair has the ability to record decisions in the absence of consensus: https://www.w3.org/policies/process/#managing-dissent Where consensus can be achieved, that's not what they should do, but we're not blocked by frivolous dissent
* If formal objections are indeed registered during the charter refinement phase (regardless of by who), that doesn't block the work on charter refinement, and gets processed later, together with any FO filed during the AC Review anyway, so that too does not create delays
* If we were to restrict the consensus body to W3C Members only, or AC Reps only, that would create undesirable friction for CGs trying to graduate into a working group. Here too, the Chair has the ability to declare group decisions even in the absence of consensus if broad participation leads to non-constructive comments, but narrow participation would risk limit discussion too much. Experience with public charter discussions in https://github.com/w3c/charter-drafts/ show that keeping this discussion open to the general public does not (at least in the general case) result in the conversation being derailed, so we can default to being open, and let the chairing facilitator be the judge of occasional cases of non-constructive dissent. If they were to abuse their power to silence constructive feedback, that's what Formal Objections are for (and the require disposition of comments would give visibility to such cases).

In terms of experimenting, landing a PR into the Process's Editor's Draft does not make it official. The Team absolutely can work from this ED to run it as an experiment, and give feedback based on that, the difference being that as long as it is an experiment, the MUSTs aren't binding, and the delayed processing of FOs doesn't automatically apply (though the Team can requests leeway from the Council).

With all that said, if there are additional concerns, you should feel free to open follow up issues, but for now, the Process CG, given the [decision of the AB to adopt this concept at a high level](https://github.com/w3c/AB-memberonly/issues/224#issuecomment-2248326969), thinks that landing the current text is the best way to give it visibility and solicit feedback on it.